test/unit/check/arbitrary/IntegerArbitrary.spec.ts

```diff
@@ -146,6 +146,9 @@ describe('IntegerArbitrary', () => {
           return -log2(-min) <= g && g <= log2(max);
         })
       ));
+    it('Should throw when minimum number is greater than maximum one', () => {

```

Comment:
Suggestion, a property based test:

```js
    it('Should throw when minimum number is greater than maximum one', () =>
      fc.assert(
        fc.property(fc.integer(), fc.integer(), (a, b) => {
          fc.pre(a !== b);
          if (a < b) expect(() => integer(b, a)).toThrowError();
          else expect(() => integer(a, b)).toThrowError();
        })
      ));
```

This property detected the issue reported for: a = 1 and b = 0